src/sgame/sg_main.cpp

```diff
@@ -926,6 +926,7 @@ G_ShutdownGame
 */
 void G_ShutdownGame( int restart )
```

Comment:
void G_ShutdownGame( int /* restart */ ) ? Then you can delete the Q_UNUSED below.